test/unit/cute/intel_xe/copy_1d.cpp

```diff
@@ -30,18 +30,21 @@
  *
  **************************************************************************************************/
 
-#include "cutlass/util/print_error.hpp"
-#include "cutlass_unit_test.h"
-
 #include <cute/tensor.hpp>
 #include <sycl/sycl.hpp>
 #include <syclcompat.hpp>
+#include <syclcompat/launch_policy.hpp>
+
+#include "cutlass_unit_test.h"
 
 using namespace cute;
 using namespace cutlass;
+using namespace syclcompat::experimental;
+
+#define SUBGROUP_SIZE (16)
 
 template <class TensorS, class TensorD>
-void copy_kernel_vectorized(TensorS tile_S, TensorD tile_D) {
+void copy_kernel_vectorized(TensorS tile_S, TensorD tile_D, void *) {
```

Comment:
why is an extra `void *` parameter added here(and in other tests), when it's not being used ?

Reply:
I notice that it seems for smem_ptr

Reply:
You do not need to pass the `void*` if you are not specifying the local memory size to the sycl compat launch

Reply:
I have no clue how to remove this parameter.   smem is allocated in my kernel not a **extern** parameter

Reply:
> I have no clue how to remove this parameter. smem is allocated in my kernel not a extern parameter

You need not pass the `local_mem_size` property to the syclcompat launch. 

> `local_mem_size{}`

Just remove the above line